### PR TITLE
add template page for careers in tech

### PR DESCRIPTION
**Pull request by tanyaparker**
The content isn't final. Just creating the template for Sarah to keep working on.

image

### CODE REVIEW

File: pegasus/sites.v3/code.org/public/careers-in-tech.haml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    %h2 Jasmine from Zillow
+    = view :display_video_thumbnail, id: 'jasmine_zillow', video_code: 'i2KHYUhtOlM', caption: 'Meet Jasmine: Jasmine is a security software engineer at Zillow, a website that helps people buy and sell houses.', play_button: 'center'
+    %a{href: '#'} Learn more
+  .col-33.soft-padding
+    %h2 Jerome from Microsoft
+    = view :display_video_thumbnail, id: 'jerome_microsoft', video_code: 'G8hfAk4PfOM', caption: 'Meet Jerome: Jerome is a Program Manager at Microsoft and works on Windows.', play_button: 'center'
+    %a{href: '#'} Learn more
+  .col-33.soft-padding
+    %h2 Brendan from Code.org
+    = view :display_video_thumbnail, id: 'brendan_codeorg', video_code: 'RQ6GJt9f2vg', caption: 'Meet Brendan: Brendan is a designer at Code.org. As a designer, Brendan helps define the look and feel of the site.', play_button: 'center'
```
Comment (breville) [inline pegasus/sites.v3/code.org/public/careers-in-tech.haml:32]: is this a placeholder?  i seem to be talking about CS concepts more than design :)

File: pegasus/sites.v3/code.org/views/jumbotron_banner.haml [hunk 2]
```diff
@@ -1,3 +1,4 @@
+- specs ||= nil
```
Comment (breville) [inline pegasus/sites.v3/code.org/views/jumbotron_banner.haml:1]: do we use this specs parameter in this PR?  or what's it for?

File: pegasus/sites.v3/code.org/public/careers-in-tech.haml (hunk 1 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/public/careers-in-tech.haml:32]: Yep! These videos are just placeholders.

File: pegasus/sites.v3/code.org/views/jumbotron_banner.haml (hunk 2 above)
Comment (tanyaparker) [reply pegasus/sites.v3/code.org/views/jumbotron_banner.haml:1]: It used to be required in jumbotron banner but I don't need it for this page, so made it optional.

**Comment by breville** [comment]
lgtm

**Pull request merged**